public/javascripts/components/SideBar.js

```diff
@@ -0,0 +1,88 @@
+"use strict";
+
+import React from "react";
+import Chance from "chance";
+import $ from "jquery";
+const chance = new Chance();
```

Comment:
Rather than using this so much, you should really try and figure out why this happens